frank/tests.py

```diff
@@ -346,6 +347,10 @@ def test_pipeline_no_phase():
     """Check the full fit pipeline when fitting for the disc's inc, PA"""
     _run_pipeline('gaussian', False)
 
+def test_pipeline_no_inc_no_pa():
```

Comment:
I can't see where fit_inc_pa is used in `_run_pipeline`

Reply:
good catch! I left out the line. Added in 79a2cab